client/client.manifest.template

```diff
@@ -21,7 +21,12 @@ loader.log_level = "{{ log_level }}"
 # applies. Paths must be in-Gramine visible paths, not host paths (i.e.,
 # paths must be taken from fs.mounts[...].path, not fs.mounts[...].uri).
 loader.env.LD_LIBRARY_PATH = "/lib:{{ arch_libdir }}:/usr/lib:/usr/{{ arch_libdir }}"
-loader.env.DEBUG = "1"
+
+loader.env.DEBUG = { passthrough = true }
+loader.env.MRENCLAVE = { passthrough = true }
+loader.env.MRSIGNER = { passthrough = true }
+loader.env.ISV_PROD_ID = "{{ isvprodid }}"
+loader.env.ISV_SVN = "{{ isvsvn }}"
```

Comment:
I think we should pass these through as well. What is happening here is, we provide the isvprodid and isvsvn of the client enclave to use these values to attest the report of the server enclave, which potentially may have different values.